### DISCUSSION TITLE
How 2 add Prism 2 Uno in a stone cold minimalist way ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#5434] [Q&A]
Very basic thing again. I have this App.cs/xaml, blank project, want 2 compile only. Hit F5 in vain. 

![image](https://user-images.githubusercontent.com/43340959/110659404-d8733700-81c2-11eb-9316-ac11b8039828.png)

![image](https://user-images.githubusercontent.com/43340959/110659434-de691800-81c2-11eb-9d2d-5aaafc83749f.png)

What shall I do to compile a `PrismApplication` ?

**Comment by jeromelaban** [comment]
Thanks for the feedback. If you want to use Prism, you can use a template from the command line [available here](https://platform.uno/docs/articles/get-started-dotnet-new.html#uno-platform-blank-prism-application), or use [Prism's extensions](https://marketplace.visualstudio.com/items?itemName=BrianLagunas.PrismTemplatePack).
